.github/workflows/ci.yml

```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        ember-try-scenario: [ember-release, ember-beta]
+        ember-try-scenario: [ember-lts-4.8, ember-release, ember-beta]
```

Comment:
can you add ember-lts-4.12 as well?

also, do we have a minimum version?

Reply:
👍🏻 - adding 4.12
- minimum version? -  (I picked 4.8-lts based on https://emberjs.com/releases/lts/, but happy to add whichever versions we want to certify.  In theory, it still should work for ember 3.x with these changes.).

Reply:
I think if we can add a 3.28 test, that'd be great for folks wanting to upgrade -- there are a number of folks who are still trying to work through deprecations stuck on 3.28

Reply:
Some sort of error about `this.project.bowerDependencies` being undefined from ember-source: https://github.com/emberjs/ember.js/blob/v3.28.11/lib/index.js#L48

I don't know what `this.project` is.

Reply:
that's just a warning, we can ignore that -- that thing has had issues for some time and is unrelated to anything we can do here

Reply:
you may need to add ember-cli in there, too

Reply:
I think I found out what the issue is?

The version of ember-cli@5.1.0 is incompatible with ember-source@3.28.12.  In [ember-source](https://github.com/emberjs/ember.js/blob/v3.28.11/lib/index.js#L48), the node module constructor is expecting a function `bowerDependencies`.  This was previously provided by lib/broccoli/ember-app.js (I think?), but all of the bower stuff has been deprecated/removed.

Some options:
- Drop support for Ember 3 in this and make ember-page-title a major version change. (I doubt there will be many feature changes / updates to this addon due to it's narrow purpose.)

- Re-configure the test app to use an older version of ember-cli (and older ember-source versions).  (I originally updated the test app because [the tests were failing](#256).)

Reply:
using ember-cli@3.28 with ember-source@3.28 in ember-try is also perfectly reasonable -- most people are not going to be using ember-cli v5.1 with ember-source@3.28

Reply:
Ok will give that a shot!  There were test errors to still fix with that combination, but I didn’t look further into them.

Reply:
Looks like the additional test errors were because of using newer @ember/test-helpers.  Adding an older dependency for 3.28 / 4.4 fixed it.

Reply:
most excellent!